weaviate/connect/helpers.py

```diff
@@ -177,7 +178,10 @@ def connect_to_custom(
         `timeout`
             The timeout to use for the underlying HTTP calls. Accepts a tuple of integers, where the first integer
             represents the connect timeout and the second integer represents the read timeout.
-
+        `auth_credentials`
+            The credentials to use for authentication with your Weaviate instance. This can be an API key, in which case use `weaviate.auth.AuthApiKey`,
+            a bearer token, in which case use `weaviate.auth.AuthBearerToken`, a client secret, in which case use `weaviate.auth.AuthClientCredentials`
```

Comment:
do you want to change the docstrin examples to `weaviate.AuthApiKey` ? so without the double `auth`?